fix(trend_micro): update the doc (#2946)
SEKOIA-IO/documentation: docs/integration/categories/email/trend_micro_email_security.md

## Configure

> For the configuration part, there are two methods because Trend Micro Email Security is migrating to Trend Vision One within the same product. For those still using the old solution, there is the first configuration method, and for those using the new one, there is a second method.

Reviewer: I propose an enhanced version:

```suggestion
This section will guide you through the creation of the API key.

According to the solution you use, the procedure will change:
- The first method is for those still using Trend Micro Email Security
- the second method is for those using Trend Vision One

```

> ### How to create an API token ( Trend Micro Email Security )

Reviewer: I suggest using tab to separate the methods

```suggestion
### How to create an API token 

=== Trend Micro Email Security
```
 +indetation for the content

1. Log on to the Trend Micro Email Security administrator console.
2. Go to **Administration**

    ![Step 1](/assets/integration/email/trend_micro/step_1.png)

3. Click on **Service Integration**. The API Access tab displays by default.

    ![Click on Service Integration](/assets/integration/email/trend_micro/step_2.png)
    
4. Click **Add** to generate an API Key.

    ![Click on Add](/assets/integration/email/trend_micro/step_3.png)

5. Copy the API Key value and save the value. Keep the API Key private

> ### How to create an API token ( Trend Vision One )

Reviewer: To be consistent with the previous comment

```suggestion
=== Trend Vision One 
```

+ Indentation for the content
